src/map_object.c

```diff
@@ -287,7 +374,7 @@ void sub_0205E580(MapObjectManager *manager) {
     LocalMapObject *objects = MapObjectManager_GetObjects(manager);
 
     do {
-        if (MapObject_IsInUse(objects) == TRUE && MapObject_GetID(objects) == 0xFF) { // todo: const (is this invalid ID? or just max?)
+        if (MapObject_CheckActive(objects) == TRUE && MapObject_GetID(objects) == 0xFF) { // todo: const (is this invalid ID? or just max?)
```

Comment:
This is probably `obj_player`

Reply:
possible, however I would rather have either an existing header for consts, or if there's a new one needed, valued for the other consts
also I'm not 100% sure about it, and I would like to be